cfme/cloud/keypairs.py

```diff
@@ -184,6 +185,41 @@ def download_private_key(self):
         view.toolbar.configuration.item_select('Download private key')
         view.flash.assert_no_error()
 
+    def set_ownership(self, owner=None, group=None, click_cancel=False, click_reset=False):
```

Comment:
This looks like something that a mixin implementation would be appropriate for, like with the Tagging classes.

We have multiple entities that can have ownership set, and they use a common form and form data. We can define an ownership class that can be inherited by any entity classes that support ownership.

I'd call it out of scope for this PR, but would be a good enhancement so we don't continue to re-implement this method on multiple entities.